lms/urls.py

```diff
@@ -1055,3 +1055,8 @@
 urlpatterns += [
     path('api/notifications/', include('openedx.core.djangoapps.notifications.urls')),
 ]
+
+# Content tagging
+urlpatterns += [
+    path('api/tagging/', include(('openedx.features.content_tagging.urls', 'content_tagging'), namespace='content_tagging')),
```

Comment:
```suggestion
    path('api/content_tagging/', include(('openedx.features.content_tagging.urls', 'content_tagging'), namespace='content_tagging')),
```

I've updated the issue too -- down the track, we may have "user_tagging" too, so better to make this consistent now.